web_app_drawer/static/src/xml/web_app_drawer.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<templates>
+
+	<t t-name="web_app_drawer.SearchField">
+		<form class="form-inline app-drawer-search-form">
+			<div class="form-group">
+				<label class="sr-only" for="appDrawerSearch">
+					Search
+				</label>
+				<input type="text"
+					   class="form-control pull-right app-drawer-search-input"
+					   id="appDrawerSearch"
+					   placeholder="Search"
+					   />
```

Comment:
Please configure your editor to use either spaces or tabs, but don't mix them. OCA and PEP8 standards seem to be 4 spaces.